### DISCUSSION TITLE
希望解释saved-props.json是什么及其作用

### DISCUSSION THREAD

**Discussion by Pr0pHesyer** [hooke007/mpv_PlayKit#367] [Q&A 问答] [answered]
在V5中，glsl-shaders(-append)完全不由mpv.conf控制，在mpv.conf中更改为无效更改。
只有在saved-props.json中更改才会生效。
这造成很大的麻烦

**Comment by hooke007** [comment]
先仔细阅读说明

**Reply by Pr0pHesyer** [reply]
根据[说明](https://hooke007.github.io/unofficial/mpv_start.html#id18)，这是主设置文件mpv.conf控制的，与实际冲突。
并且说明完全没有提及saved-props.json。
请你解释说明与实际的问题

**Reply by hooke007** [reply, marked as answer]
> 懒人包是解压即可使用的完整解决方案
使用前务必完整阅读 https://github.com/hooke007/MPV_lazy/discussions/194

你看的是mpv的使用介绍，不是懒人包的说明

另外在下载的地方我还着重标注了
必读
必读
必读
<img width="498" alt="截屏2023-09-17 16 48 11" src="https://github.com/hooke007/MPV_lazy/assets/41094733/ac930b29-d43b-407e-8fd6-f668e767a189">

**Reply by Pr0pHesyer** [reply]
抱歉，没有搜索文档是我的疏忽。
基于该文档及应用行为，希望能改善用户体验。

1.取消预设退出时记住着色器列表。
退出时记住着色器列表，那么mpv.conf里的着色器列表/着色器追加只在第一次生效，之后则会被saved-props.json中的覆盖而不生效。
从我个人来说，修改mpv.conf以改变着色器是最直接也最符合直觉的，并且各文档都以mpv.conf为主，修改mpv.conf以改变应用选项。
2.修改文档及注释
若保留退出时记忆着色器列表，则希望能在mpv.conf里添加注释，说明着色器列表默认在退出时记忆且会在应用启动时覆盖mpv.conf的选项。

**Reply by hooke007** [reply]
有没有一种可能，这个当前行为就是根据用户反馈创造的需求，并且成为当前的默认行为。

另外置顶帖已有意见征集，此类观点数量不足（因为没人反馈过

**Reply by Pr0pHesyer** [reply]
即便是用户反馈，您才是项目管理员，具体由您掌控。
我仍然会使用该项目，但是若保留退出时记忆着色器列表，则希望能在mpv.conf里着色器部分添加注释，说明着色器列表默认在退出时记忆且会在应用启动时覆盖mpv.conf的选项。
我会在置顶帖中回复，并且该讨论可关闭
